index.html

```diff
@@ -0,0 +1,937 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
+        integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+        <link rel="stylesheet" href="assets/css/reset.css">
+    <link rel="stylesheet" href="assets/css/style.css">
+</head>
+
+<body>
+    <div class="container main-container">
+        <header class="header">
+            <div class="row main-header">
+                <div class="col-md-4 date">
+                    <span>Wed, Mar 27, 2019</span>
+                </div>
+                <div class="col-md-4">
+                    <span class="site-title">NewsPaper</span>
+                </div>
+                <div class="col-md-4 signup">
+                    <ul>
+                        <li>SIGN IN</li>
+                        <li>SUBSCRIBE ></li>
+                    </ul>
+                </div>
+            </div>
+            <nav class="nav">
+                <ul class="main-nav">
+                    <li>U.S</li>
+                    <li>World</li>
+                    <li>Business</li>
+                    <li>Tech & Science</li>
+                    <li>Culture</li>
+                    <li>Sports</li>
+                    <li>Health</li>
+                    <li>Opinion </li>
+                    <li>Ndelible</li>
+                    <li><input type="text"></li>
+                </ul>
+            </nav>
+        </header>
+        <div class="row">
+            <div class="col-md-4">
+                <div class="">
```

Comment:
Remove empty `class` attributes and also on line 74,84,95,105 and so on

Reply:
fixed